### PR TITLE
tests/doc: Fixed group names and docs required of some modules

### PR DESCRIPTION
**Pull request by eduazocar**
### Contribution description
This new PR organize all test group for the documentaton. Was fixed renaming all the test doxygen groups. this adds some file that didn't exist in some test files. 

### Testing procedure

run make docs:
```
make docs
```

### CODE REVIEW

File: examples/doc.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
-@defgroup examples Mesh4all Examples
+@defgroup examples Examples
```
Comment (luisan00) [inline examples/doc.txt:2]:
```suggestion
@defgroup examples Sample applications running on some of our custom boards
```

File: examples/ds18b20/doc.txt [hunk 2]
```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @defgroup    ds18b20 Ds18b20
+ * @defgroup    examples_ds18 Ds18b20
```
Comment (luisan00) [inline examples/ds18b20/doc.txt:10]:
```suggestion
 * @defgroup    examples_ds18 Sample application using a ds18 temperature sensor
```

File: examples/ds18b20/doc.txt [hunk 3]
```diff
@@ -38,7 +38,7 @@ You could change the ds18b20 params to get another gpio controller.
 
 If you want to upload the example to your board, needs to compile first, additional to this you could initialize the terminal and watch the results
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-make -C examples/esp32-wroom-32/test_ds18b20 flash term
+make -C examples/ds18b20 flash term
```
Comment (luisan00) [inline examples/ds18b20/doc.txt:41]:
```suggestion
make -C examples/ds18 flash term
```
also the folder name should be `examples/ds18`

File: examples/ds18b20/main.c [hunk 4]
```diff
@@ -15,7 +15,7 @@
  */
 
 /**
- * @ingroup     ds18b20
+ * @ingroup     examples_ds18b20
```
Comment (luisan00) [inline examples/ds18b20/main.c:18]:
```suggestion
 * @ingroup     examples_ds18
```

File: examples/external_led/doc.txt [hunk 5]
```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @defgroup    external-led External-led blink
+ * @defgroup    examples_external-led External-led blink
```
Comment (luisan00) [inline examples/external_led/doc.txt:10]:
```suggestion
 * @defgroup    examples_blinking_led Blinking LEDs 
```

Comment (luisan00) [reply examples/external_led/doc.txt:10]: or  just `blinking`, `blink-led`, etc

File: examples/external_led/doc.txt [hunk 6]
```diff
@@ -55,7 +55,7 @@ another aspect to be take into account is add the xtimer and ztimer headers file
 
 If you want to upload the example to your board, needs to compile first, additional to this you could initialize the terminal and watch the results
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-make -C examples/esp32-wroom-32/external_led flash term
+make -C examples/external_led flash term
```
Comment (luisan00) [inline examples/external_led/doc.txt:58]: here the same

File: examples/external_led/main.c [hunk 7]
```diff
@@ -15,7 +15,7 @@
  */
 
 /**
- * @ingroup     examples_esp32-wrover
+ * @ingroup     examples_external-led
```
Comment (luisan00) [inline examples/external_led/main.c:18]: same

File: examples/mesh4all_rpl_dag/doc.txt [hunk 8]
```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2022 Mesh4all <mesh4all.org>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    examples_rpl_dag External-led blink
```
Comment (luisan00) [inline examples/mesh4all_rpl_dag/doc.txt:10]:
```suggestion
 * @defgroup    examples_rpl_dag External-led blink
```
wrong group here

Comment (luisan00) [reply examples/mesh4all_rpl_dag/doc.txt:10]: also wrong description, External-led-blink?

File: examples/moisture_sensor_hw390/doc.txt [hunk 9]
```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @defgroup    moisture_sensor_example  Analog capacitive moisture sensor
+ * @defgroup    examples_moisture_sensor  Analog capacitive moisture sensor
```
Comment (luisan00) [inline examples/moisture_sensor_hw390/doc.txt:10]:
```suggestion
 * @defgroup    examples_moisture_sensor  Sample application using a cheap analog capacitive moisture sensor
```

File: examples/slipdev/doc.txt [hunk 10]
```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- @defgroup    slipdev Slipdev
+ @defgroup    examples_slipdev Slipdev
```
Comment (luisan00) [inline examples/slipdev/doc.txt:10]:
```suggestion
 @defgroup    examples_slipdev Slipdev
```
something more descriptive here could helps

File: examples/uart_hc-06/README.md [hunk 11]
```diff
@@ -16,5 +14,5 @@ to send messages through uart to the bluetooth device write `send message` since
 to compile this example:
 
 ```sh
-make -C examples/tpm_uart  BOARD=your_board  PORT=/dev/your_port flash term
+make -C examples/uart_hc-06  BOARD=your_board  PORT=/dev/your_port flash term
```
Comment (luisan00) [inline examples/uart_hc-06/README.md:17]: this example should be deleted from the main branch

File: tests/border_router/doc.txt [hunk 12]
```diff
@@ -1,5 +1,6 @@
-@defgroup border_router_test border router test module
-@ingroup     test_group
+/**
+@defgroup    tests_border_router Border router test
```
Comment (luisan00) [inline tests/border_router/doc.txt:2]: add something more descriptive here too

File: tests/chamos_test/doc.txt [hunk 13]
```diff
@@ -1,5 +1,5 @@
-@defgroup chamos_test chamos test module
-@ingroup     test_group
+@defgroup    tests_chamos chamos test
```
Comment (luisan00) [inline tests/chamos_test/doc.txt:1]: and here

File: tests/doc.txt [hunk 14]
```diff
@@ -1,5 +1,5 @@
 /**
-@defgroup test_group Test
+@defgroup tests Tests
```
Comment (luisan00) [inline tests/doc.txt:2]: and here

File: tests/radio/doc.txt [hunk 15]
```diff
@@ -1,5 +1,5 @@
-@defgroup Radio_test Radio test module
-@ingroup     test_group
+@defgroup   tests_radio Radio test
```
Comment (luisan00) [inline tests/radio/doc.txt:1]: description here also could helps

File: tests/rpl/doc.txt [hunk 16]
```diff
@@ -0,0 +1,26 @@
+
+/**
+@defgroup   tests_rpl RPL test
```
Comment (luisan00) [inline tests/rpl/doc.txt:3]: same

**Review by luisan00 — CHANGES_REQUESTED** [review]

File: examples/doc.txt (hunk 1 above)
Comment (eduazocar) [reply examples/doc.txt:2]: I add as description a brief documentation parameter, Could this works to a brew description or should take the suggested and apply as title?

**Comment by luisan00** [comment]
Looks good, only do a squash and ready to go ahead

**Comment by eduazocar** [comment]
Ok, squash is already :+1:

**Review by luisan00 — APPROVED** [review]
Good job 👏

**Pull request merged**